assets/dev/js/frontend/utils/lightbox/lightbox.js

```diff
@@ -451,18 +542,34 @@ module.exports = elementorModules.ViewModule.extend( {
 	activateFullscreen: function() {
 		const classes = this.getSettings( 'classes' );
 		screenfull.request( this.elements.$container.parents( '.dialog-widget' )[ 0 ] );
-		this.elements.$iconExpand.removeClass( classes.slideshow.iconExpand )
-			.addClass( classes.slideshow.iconShrink )
-			.attr( 'aria-checked', 'true' );
+
+		if ( elementorFrontend.config.experimentalFeatures.e_font_icon_svg ) {
+			this.elements.$iconExpand.addClass( classes.hidden ).attr( 'aria-checked', 'true' );
+			this.elements.$iconMinimize.removeClass( classes.hidden ).attr( 'aria-checked', 'true' );
```

Comment:
Check why do you need to add 'aria-checked' for both elements

Reply:
Fixed (also added to the zoom buttons that were missing in some cases).